app/views/contexts/view_preview_questions_context.py

```diff
@@ -0,0 +1,38 @@
+from datetime import datetime
+from typing import Union
+
+from flask import url_for
+
+from app.data_models import QuestionnaireStore
+from app.questionnaire.questionnaire_schema import QuestionnaireSchema
+from app.views.contexts.preview_context import PreviewContext
+
+
+def build_view_preview_questions_context(
+    language: str,
+    schema: QuestionnaireSchema,
+    questionnaire_store: QuestionnaireStore,
+) -> dict[str, Union[str, Union[str, datetime], dict]]:
+
+    metadata = questionnaire_store.metadata
+
+    context = {
+        "hide_sign_out_button": True,
+        "metadata": metadata,
+        "submitted_text": "",
```

Comment:
Are these two needed?

Reply:
Not needed after we refactored the classes, removed.